### DISCUSSION TITLE
Syncronised tabset

### DISCUSSION THREAD

**Discussion by rgaiacs** [quarto-dev/quarto-cli#2039] [Feature Requests]
Some tutorial or documentation cover two or more languages using [tabset](https://quarto.org/docs/interactive/layout.html#tabset-panel). For example, https://framework.frictionlessdata.io/docs/guides/quick-start has examples for `CLI`, `Python`, and `API`. A minimal working example is

```
---
title: "Minimal Working Example"
---

Some text

::: {.panel-tabset}

## Python

    x = 1

## R

    x <- 1

:::

More text

::: {.panel-tabset}

## Python

    y = 2

## R

    y <- 2

:::
```

Can we add a small JavaScript and YAML option to syncronise all tabsets? For example, when the reader click in the `R` tab in the first code sample, the second code sample also change.

**Comment by jjallaire** [comment]
We have an extension available for this now: https://github.com/quarto-ext/grouped-tabsets

The feature is also built in to Quarto v1.1 (pre-release here: https://quarto.org/docs/download/prerelease.html)

**Comment by mcanouil** [comment]
Closing as completed.
